src/js/index.js

```diff
@@ -112,32 +116,24 @@ const tiles = L.tileLayer(config.tiles, {
 
 new L.Hash(map);
 
-// define query object
-const query = {
-	no_overlay: "false"
-};
-// iife function to read query parameter and fill query object
-(function () {
-	let telem;
-	const search_values = location.search.replace('\?', '').split('&');
-	for (let i = 0; i < search_values.length; i++) {
-		telem = search_values[i].split('=');
-		query[telem[0]] = '';
-		if (typeof telem[1] != 'undefined') query[telem[0]] = telem[1];
+const query = querystring.parse(window.location.search.substring(1));
+if (query.center) {
+	//Coordinates are passed by query
+	// /?center={lat},lng&zoom={zoom}
+	const center = query.center.split(',').map(coord => parseFloat(coord));
+	coordsCenter = center;
+	if (query.zoom) {
+		zoomLevel = parseInt(query.zoom);
 	}
-})();
-
-// show betterplace overlay
-if (query.no_overlay === "false") d3.select("#betterplace").style("display", "inline-block");
-
-let coordsCenter = config.center;
-let zoomLevel = config.zoom;
-
-if (location.hash) {
+} else if (location.hash) {
+		// Coordinates are passed by hash path
+	  // /#{zoom}/{lat}/{lng}
 	const hash_params = location.hash.split("/");
 	coordsCenter = [hash_params[1], hash_params[2]];
 	zoomLevel = hash_params[0].substring(1);
```

Comment:
Why are we using our own center and zoom instead of using hashpaths similar to here?

Reply:
Hi @DavidLemayian We are catering for query instances for when the map is on the site like such https://github.com/CodeForAfrica/sensors.AFRICA/blob/master/src/components/SensorMap/index.js and here as well https://github.com/CodeForAfrica/sensors.AFRICA/blob/master/src/components/Embeds/AirMap.js 
Then again I could have missed something here that we could have used instead, like the hashpaths